src/forms/UncontrolledTextarea.jsx

```diff
@@ -0,0 +1,67 @@
+// @flow
+import * as React from 'react';
+import Textarea from './Textarea';
+
+type State = {
+	value: string,
+};
+
+type Props = {
```

Comment:
should these props match the props for `Textarea`?

Reply:
That could be useful. I wasn't sure if I should explicitly declare them since this returns `<Textarea />` and they aren't used in this component aside from being passed through. Is it conventional to declare all possible props that might be used?

Reply:
all the props that are supported, yeah - consumer components shouldn't have to be aware of the implementation details, they just need to know what they can pass.

I recommend importing the `Props` type from `Textarea` and using it here.